src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -975,15 +975,15 @@ private static void SetLastError(WriteEventErrorCode error)
         // <UsesUnsafeCode Name="Local v7 of type: Char*" />
         // <ReferencesCritical Name="Method: EncodeObject(Object&, EventData*, Byte*):String" Ring="1" />
         // </SecurityKernel>
-        internal unsafe bool WriteEvent(ref EventDescriptor eventDescriptor, IntPtr eventHandle, Guid* activityID, Guid* childActivityID, params object?[] eventPayload)
+        internal unsafe bool WriteEvent(ref EventDescriptor eventDescriptor, IntPtr eventHandle, Guid* activityID, Guid* childActivityID, object?[] eventPayload)
```

Comment:
Is there anywhere that uses the `param` functionality that might break as a result of this?

Reply:
There's only two call sites to this method, both passing an array rather than relying on params, and there's no other overload they could inadvertently bind to.  The params was just superfluous.  If I'm missing something, please let me know :-)

Reply:
Not that I'm aware of. Just wanted to be thorough :)